UI/Program.cs

```diff
@@ -117,6 +117,30 @@
 
 app.MapRazorPages();
 
+app.Use(async (context, next) =>
+{
+    if (!context.Response.Headers.ContainsKey("X-Frame-Options"))
+    {
+        context.Response.Headers.Add("X-Frame-Options", "DENY");
+    }
+
+    if (!context.Response.Headers.ContainsKey("Content-Security-Policy"))
+    {
+        context.Response.Headers.Add("Content-Security-Policy", "base-uri 'self'; block - all - mixed - content; default - src 'self'; img - src data: https:; object-src 'none'; script - src 'self'; style - src 'self'; upgrade - insecure - requests; ");
```

Comment:
There seem to be extra spaces around the hyphens here. Looking at [the example](https://docs.microsoft.com/en-us/aspnet/core/blazor/security/content-security-policy?view=aspnetcore-6.0#blazor-server) they should all be kebab case without spaces

Reply:
Spaces removed